components/public-api/gitpod/v1/workspace.proto

```diff
@@ -38,6 +43,28 @@ message WatchWorkspaceStatusResponse {
   WorkspaceStatus status = 2;
 }
 
+message ListWorkspacesRequest {
+  // pagination contains the pagination options for listing workspaces
+  PaginationRequest pagination = 1;
+
+  // organization_id is the ID of the organization that contains the workspaces
```

Comment:
mention that it is amandatory

Reply:
@mustard-mh proto does not have concept of required, it is in a guideline: https://www.notion.so/gitpod/Public-API-guideline-a010fa3d494544b78f28945e53f730ac?pvs=4#ea845138f4624b0b93ef56670c6d3085

> Therefore, we describe field behavior, such as whether it is required or optional, in the documentation and enforce it with runtime checks.

Reply:
> Therefore, we describe field behavior, such as whether it is required or optional, in the documentation and enforce it with runtime checks.

@akosyakov  I checked we didn't add any related `required` in new proto files, could we have a guideline like how to add it? Since I saw we have `+optional` in document. I will first go with `+required` first

Reply:
ok, let's try like that, we will need another round to go through our docs anyway